### DISCUSSION TITLE
Allow to compare "Name" triggers

### DISCUSSION THREAD

**Discussion by GV123h** [ikemen-engine/Ikemen-GO#926] [Feature Request]
**Is your feature request related to a problem?
Yes, every time I try to use the Name triggers they require to be compared to something enclosed in " "
but I want to compare it to another trigger example:
P1Name = P2Name

**Describe the solution you'd like**

Let the Name trigger be compared to another triggers that return text Info

**Comment by Windblade-GR01** [comment]
We have this planned for the future.
The plan is able to save strings to variables and also allow to compare strings.

**Comment by potsmugen** [comment]
This is working in my branch. Hope to push it for version 1.1.
